packages/cli/package.json

```diff
@@ -22,6 +22,7 @@
     "build": "backstage-cli build-cache -- tsc",
     "lint": "backstage-cli lint",
     "test": "backstage-cli test",
+    "clean": "backstage-cli clean",
```

Comment:
This should be added to every package that uses the backstage-cli, bonus points for updating the plugin and app templates in `packages/cli/templates/` too :grin:

Reply:
Should I add clean script into plugins packages?

Reply:
Yep!